src/org/loklak/api/admin/CampaignServlet.java

```diff
@@ -37,59 +37,75 @@
 import org.loklak.tools.DateParser;
 
 public class CampaignServlet extends HttpServlet {
-   
+
     private static final long serialVersionUID = 8578478303032749879L;
 
     @Override
     protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         doGet(request, response);
     }
-    
+
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         Query post = RemoteAccess.evaluate(request);
-        
+
         boolean localhost = post.isLocalhostAccess();
         if (!localhost) {response.sendError(400, "campaigns can only be started by calls from localhost"); return;}
```

Comment:
@singhpratyush there are few more if statements left in AccessServlet.java, CampaignServlet.java, CrawlerServlet.java,StatusService.java. Please check all the files once and correct them.